tests/tasty/Arbitrary.hs

```diff
@@ -143,34 +143,69 @@ instance Arbitrary KVar where
 instance Arbitrary GradInfo where
   arbitrary = pure $ GradInfo (SS pos pos) Nothing
     where pos = Spans.dummyPos "<unknown>"
+  shrink _ = mempty
```

Comment:
The documentation of `shrink` says the default instance is `const []`. What difference does this change make?

Reply:
The explicit implementation served as documentation when I was writing the shrinkers of which types I had written shrinkers for and in particular which types did not need a shrinker for whatever reason (since they had an explicit implementation of shrink).  However, I can remove it if you'd like.

Reply:
Added comment.